Firestore/Example/Tests/API/FSTUserDataReaderTests.mm

```diff
@@ -112,38 +124,42 @@ - (void)testConvertsUnsignedCharToInteger {
 - (void)testConvertsStrings {
   NSArray<NSString *> *values = @[ @"", @"abc" ];
   for (id value in values) {
-    FieldValue wrapped = FSTTestFieldValue(value);
-    XCTAssertEqual(wrapped.type(), FieldValue::Type::String);
-    XCTAssertEqual(wrapped.string_value(), util::MakeString(value));
+    google_firestore_v1_Value wrapped = FSTTestFieldValue(value);
+    XCTAssertEqual(GetTypeOrder(wrapped), TypeOrder::kString);
+    XCTAssertEqual(nanopb::MakeString(wrapped.string_value), util::MakeString(value));
   }
 }
 
 - (void)testConvertsDates {
   NSArray<NSDate *> *values =
       @[ FSTTestDate(1900, 12, 1, 1, 20, 30), FSTTestDate(2017, 4, 24, 13, 20, 30) ];
   for (NSDate *value in values) {
-    FieldValue wrapped = FSTTestFieldValue(value);
-    XCTAssertEqual(wrapped.type(), FieldValue::Type::Timestamp);
-    XCTAssertEqual(wrapped.timestamp_value(), MakeTimestamp(value));
+    google_firestore_v1_Value wrapped = FSTTestFieldValue(value);
+    XCTAssertEqual(GetTypeOrder(wrapped), TypeOrder::kTimestamp);
+    firebase::Timestamp timestamp = MakeTimestamp(value);
```

Comment:
Please add using declarations for `Timestamp` and `GeoPoint`.

Reply:
Done